app/components/verkiezingen/draft-mandataris-list.hbs

```diff
@@ -1,137 +1,139 @@
-<div class="au-o-box no-pagination">
-  <h1 class="au-u-h2 au-u-margin-bottom-small">{{@title}}</h1>
-  <AuDataTable
-    @content={{this.resortedMandatarissen}}
-    @noDataMessage="Geen mandatarissen gevonden"
-    @sort={{@sort}}
-    @page={{@page}}
-    @size={{@size}}
-    as |t|
-  >
-    <t.content as |c|>
-      <c.header>
-        <AuDataTableThSortable
-          @field="isBestuurlijkeAliasVan.gebruikteVoornaam"
-          @currentSorting={{@sort}}
-          @label="Voornaam"
-        />
-        <AuDataTableThSortable
-          @field="isBestuurlijkeAliasVan.achternaam"
-          @currentSorting={{@sort}}
-          @label="Familienaam"
-        />
-        {{#if @showFractie}}
+<div {{did-update this.onInit @sort}}>
+  <div class="au-o-box no-pagination">
+    <h1 class="au-u-h2 au-u-margin-bottom-small">{{@title}}</h1>
+    <AuDataTable
+      @content={{this.resortedMandatarissen}}
+      @noDataMessage="Geen mandatarissen gevonden"
+      @sort={{@sort}}
+      @page={{@page}}
+      @size={{@size}}
+      as |t|
+    >
+      <t.content as |c|>
+        <c.header>
           <AuDataTableThSortable
-            @field="heeftLidmaatschap.binnenFractie.naam"
+            @field="isBestuurlijkeAliasVan.gebruikteVoornaam"
             @currentSorting={{@sort}}
-            @label="Fractie"
+            @label="Voornaam"
           />
-        {{/if}}
-        {{#if @showFunctie}}
           <AuDataTableThSortable
-            @field="bekleedt.bestuursfunctie.label"
+            @field="isBestuurlijkeAliasVan.achternaam"
             @currentSorting={{@sort}}
-            @label="Mandaat"
+            @label="Familienaam"
```

Comment:
Should we start with translation files so we have the same column name for every table when we want to display the firstname?  This is ok but just a question 😁

Reply:
no we won't do translation files